### PR TITLE
fix: MacOS VSCode can't load proper `appDir`

### PR DESCRIPTION
**Pull request by zent1n0**
image

### CODE REVIEW

**Comment by RimuruChan** [comment]
Fixed: #3

File: extension.js
Comment (zent1n0) [inline extension.js:1]: Considering use `const appDir = path.join(vscode.env.appRoot, 'out')` instead...

**Review by zent1n0 — COMMENTED** [review]
@gitiy1 please help with testing on multiple platforms

File: extension.js
Comment (gitiy1) [reply extension.js:1]: Up to commit 29448b46471729663cc885c0fafcf180e919b79b and version 1.2.0, this extension works right on my device and now i haven't met any errors. It seems like the arg change also works for me. Thanks a lot for your guys hard work. this extension 高性能ですから

**Pull request merged**